corehq/apps/app_manager/detail_screen.py

```diff
@@ -138,7 +138,7 @@ def variables(self):
     @property
     def template(self):
         template = sx.Template(
-            text=sx.Text(xpath_function=self.xpath_function),
+            text=(sx.Text(xpath_function=self.xpath_function) if self.xpath_function else sx.Text()),
```

Comment:
Note: The text is the column value for the row in the case list. It was anyway redundant since sort calculation would be calculated and displayed.

This is different from the "Display" value set for the sort calculation in app manager. That is still applicable and will be used to list the sort options on the mobile app.